CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ endif()
 
 # For some reason vtk find errors because it is not attempting to find, but referencing
 # JsonCpp, when my temporary prefix is used lol
-find_package(JsonCpp REQUIRED)
+# find_package(JsonCpp REQUIRED)
```

Comment:
this shit needs to be fixed. on macos for inscrutable reasons, vtk fails to be found without this.  either vtk macos version is buggy (but why does it work in autocrane), or some fuckery is going on.

Reply:
apparently an opencv cmake problem